src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,162 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(value);
```

Comment:
When creating a new `Node<T>`, pass all the fields to the `Node` constructor. Currently, only `value` is being passed, but `prev` and `next` should also be included if they are known at the time of node creation.